File: server/models/Monthly_data.js
import mongoose from "mongoose";
const { Schema } = mongoose;

const MonthlyEmployeeDataSchema = new Schema(
  {
    month: {
      type: Number, 
      required: true,
    },
    year: {
      type: Number,
      required: true,
    },
    employees:{
      type:Object,
      employeeId: {
          type: String,
      },
      name: {
          type: String,
      },
        email: {
          type: String,
      },
      contact: {
          type: Number,
      },
      dailyFoodCount: {
          type: Number,
      },
  },
  },
  { timestamps: true }
);

export default mongoose.model('MonthlyEmployeeData', MonthlyEmployeeDataSchema);

File: Flask/models/Monthly_data.py
from mongoengine import Document, IntField, ListField

class MonthlyEmployeeData(Document):
    month = IntField(required=True)
    year = IntField(required=True)
    employees = ListField(required=True)

    def __repr__(self):
        return f"MonthlyEmployeeData('{self.month}', '{self.year}')"
